spec/unit/shoryuken/error_handler_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+require 'airbrake/shoryuken/error_handler'
+
+RSpec.describe Airbrake::Shoryuken::ErrorHandler do
+  let(:error) { AirbrakeTestError.new('shoryuken error') }
+  let(:body) { { message: 'message' } }
+  let(:notice) { double 'notice' }
+  let(:worker) { FooWorker.new }
+  let(:endpoint) do
+    'https://airbrake.io/api/v3/projects/113743/notices?key=fd04e13d806a90f96614ad8e529b2822'
+  end
+
+  class FooWorker; end
```

Comment:
> the issue with that is because we use worker.class.to_s, which will return something like Class:0x007fd03e57c360

I can see that problem, so that's why I suggested to redefine `to_s`. If not, we can use a regexp to match that value.

> Ideally, I think this should use worker_name, but I didn't want to add Shoryuken as a development dependency in the gemspec.

Thanks for linking to that file. I'll be useful if eventually we'll come up with some integration tests in the future (say, with Rails). We'll have to depend on Shoryuken in that case. As of now, I think the current approach will do.

> If we go back with unwrapping the name space it won't be that global anymore.

Good point, I actually never thought of it like that. That said, we still don't use that style and It would add an unwanted constant under `Airbrake::Shoryuken`, too.